code/game/jobs/job/assistant.dm

```diff
@@ -11,13 +14,48 @@
 	economic_modifier = 1
 	access = list()			//See /datum/job/assistant/get_access()
 	minimal_access = list()	//See /datum/job/assistant/get_access()
+
 	outfit_type = /decl/hierarchy/outfit/job/assistant
-	alt_titles = list("Technical Assistant","Medical Intern","Research Assistant",
-					"Visitor" = /decl/hierarchy/outfit/job/assistant/visitor,
-					"Resident" = /decl/hierarchy/outfit/job/assistant/resident)
+	job_description = "An Assistant does whatever is requested of them. Though they are part of the crew, they have no real authority."
+	alt_titles = list("Assistant" = /datum/alt_title/assistant, "Technical Assistant" = /datum/alt_title/tech_assist,
+						"Medical Intern"= /datum/alt_title/med_intern, "Research Assistant" = /datum/alt_title/research_assist,
+						"Visitor" = /datum/alt_title/visitor)
 
 /datum/job/assistant/get_access()
 	if(config.assistant_maint)
 		return list(access_maint_tunnels)
 	else
 		return list()
+
+// Assistant Alt Titles
+/datum/alt_title/assistant
+	title = "Assistant"
+	title_blurb = "An Assistant does whatever is requested of them. Though they are part of the crew, they have no real authority."

```

Comment:
I _think_ I used Assistant as my test bed, so I'll try to figure that out soonish. It may have been my attempt to implement a system to have the base alt-title of the job act as the default for every other alt title, like another comment active at the moment, but I honestly can't remember right now.